metadata-ingestion/src/datahub/ingestion/run/pipeline.py

```diff
@@ -186,11 +186,13 @@ def run(self) -> None:
 
             if not self.dry_run:
                 self.sink.handle_work_unit_start(wu)
-            record_envelopes = extractor.get_records(wu)
-            for record_envelope in self.transform(record_envelopes):
-                if not self.dry_run:
-                    self.sink.write_record_async(record_envelope, callback)
-
+            try:
+                record_envelopes = extractor.get_records(wu)
+                for record_envelope in self.transform(record_envelopes):
+                    if not self.dry_run:
+                        self.sink.write_record_async(record_envelope, callback)
+            except Exception as e:
+                logger.debug(f"Failed to extract some records due to: {e}")
```

Comment:
Shall we change this to logger.error ?

Reply:
@mayurinehate yeah i have changed to logger.error